### DISCUSSION TITLE
Is there a way to make my images into a hexagon shape?

### DISCUSSION THREAD

**Discussion by Andrea-gli** [twilio-labs/paste#2979] [General] [answered]
Hi Team, 

I am trying to convert a legacy application on Console that was written in PHP with React and Paste. 
With Paste, i am only able to make the image into a square shape, but the legacy app has the images in hexagon shape. is there a way to preserve that shape using Paste? 

Legacy page:
![Screenshot 2023-01-24 at 4 13 46 PM](https://user-images.githubusercontent.com/62179195/214418264-01b8790f-0bb7-4380-8327-26d87652446e.png)

React page:
![Screenshot 2023-01-24 at 4 14 16 PM](https://user-images.githubusercontent.com/62179195/214418304-9c36c658-9c55-489c-836e-0d560eb0f7ed.png)

**Comment by gloriliale** [comment]
Hi @Andrea-gli! Is there a specific reason you're looking to preserve the hexagonal shape for this part of the application?

**Reply by Andrea-gli** [reply]
hi @gloriliale , thanks for getting back to me. According to our PM, preserving the shape is quite important for our Addon application branding. However, I am not too sure about the details. Let me also invite him to the conversation, he might have a better answer to it. thank you! @krisgutta

**Comment by krisgutta** [comment, marked as answer]
@gloriliale The current shape was what was designed to represent Add-ons in the Marketplace back in 2016, and that shape represents the product. So we are trying to make sure our customers have consistent experience while we migrate legacy stuff behind the scenes. Thanks.

**Reply by serifluous** [reply]
Hey @krisgutta! I chatted with Nathan S. on our Brand/Core Creative team, and he said the team doesn't have product-specific branding anymore. It might be worth revisiting whether the hexagon shape is still needed as a branding element since it was done in 2016. Do you know how much customer risk it'd be to swap it to a square, for example, and whether it'd be worth the risk to bring it in line with our current brand guidelines?

Something else to keep in mind is that companies may have guidelines that dictate a minimum amount of clearspace around their logos, which could be difficult to meet in a hexagon shape. For example, our own logo's guidelines requires space around it that's at least half the logomark's size: https://www.twilio.com/brand/elements/logos#:~:text=one%2Dcolor%20environments-,Logo%20clearspace,-Clearspace%20is%20the
